flake.nix

```diff
@@ -250,6 +251,9 @@
                   hooks = {
                     fourmolu.enable = true;
                   };
+                  tools = {
+                    fourmolu = p.tool "fourmolu" fourmoluVersion;
```

Comment:
@neilmayhew let me know if you have something better. It would be good to `inherit` `fourmolu` from the `tools` in `cabalProject` somehow.

Reply:
I don't think there's any way to do it